CONTRIBUTING.md

```diff
@@ -21,7 +21,6 @@ export FLATPAK_ENABLE_SDK_EXT=node14
 
 5. Within the integrated terminal of your editor, execute the following commands at the root of the repository:
 ```bash
-npm install --global yarn
 yarn install
```

Comment:
I guess we can just recommend npm nowadays?

Reply:
/me still prefers yarn over npm

Reply:
It seems to be installed with the [SDK](https://github.com/flathub/org.freedesktop.Sdk.Extension.node18/blob/cfdcf070731c5b1914af08ad1b97564e5a8daa97/org.freedesktop.Sdk.Extension.node18.yaml#L41)

Reply:
> I trust you guys with the recommendations, but don’t you have to install yarn first? Or is it nowadays packaged with npm?

this is the instructions for developing with a flatpaked vscode